file_snapshot.go

```diff
@@ -240,7 +240,11 @@ func (f *FileSnapshotStore) getSnapshots() ([]*fileSnapshotMeta, error) {
 		// Ignore any temporary snapshots
 		dirName := snap.Name()
 		if strings.HasSuffix(dirName, tmpSuffix) {
-			f.logger.Printf("[WARN] snapshot: Found temporary snapshot: %v", dirName)
+			f.logger.Printf("[WARN] snapshot: Found temporary snapshot: %v, deleting..", dirName)
```

Comment:
I think that would work and makes sense. I'd double check what context the `Close()` gets called on to ensure that it's done as part of taking a snapshot. I think it is, otherwise reap seems like it would be unsafe, too, but good to double check where that gets called.